packages/components/src/components/progress-indicator/_progress-indicator.scss

```diff
@@ -257,7 +257,8 @@
 
   .#{$prefix}--progress--vertical .#{$prefix}--progress-step,
   .#{$prefix}--progress--vertical .#{$prefix}--progress-step-button {
-    display: list-item;
+    display: block;
+    flex-direction: column;
```

Comment:
Quick-question, with `display: block` above does `flex-direction` influence this? Couldn't remember if it applied or not to flex children or if it was for containers 👀

Reply:
seems to be related to a Chrome bug https://github.com/philipwalton/flexbugs/issues/126, but I've rewritten it to avoid this